components/Content/Videos.jsx

```diff
@@ -1,14 +1,30 @@
 import { useMemo } from 'react';
 import PropTypes from 'prop-types';
 import Link from 'next/link';
+import { isNaN } from 'lodash';
+
 import videos from 'data/videos.json';
 import Video from './Video';
 
 const Videos = ({ limit }) => {
-  const videosSortedByDate = useMemo(
-    () => videos.sort((a, b) => new Date(b.date) - new Date(a.date)),
-    [],
-  );
+  const videosSortedByDate = useMemo(() => {
+    const sortedVideos = [...videos];
+
+    function isDate(date) {
+      const parsedDate = new Date(date);
+      return !isNaN(parsedDate.getTime());
+    }
+
+    const validDates = sortedVideos.filter((video) => isDate(new Date(video.date)));
```

Comment:
we do `new Date(date)` in `isDate` already, maybe can not do new Date here? and in 20th line